jib-gradle-plugin/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 ### Changed
 
 ### Fixed
+- Fixed `NullPointerException` during preconditions validation (in Java 9+) when checking for presence of `null` value in ImmutableCollection.([#2702](https://github.com/GoogleContainerTools/jib/issues/2702))
```

Comment:
Add a blank line after `### Fixed`.

The message sounds too technical and not so helpful to the end user. The CHANGELOG is for the end user where they want to quickly find relevant bug fixes and versions that matter to them.

```suggestion
- Fixed `NullPointerException` during input validation (in Java 9+) when configuring Jib parameters using certain immutable collections (such as `List.of()`). ([#2702](https://github.com/GoogleContainerTools/jib/issues/2702))
```

Reply:
This is much better! Thank you